src/components/graph/menus/root-network/root-network-panel.tsx

```diff
@@ -25,16 +27,42 @@ const styles = {
 const RootNetworkPanel: FunctionComponent = () => {
     const [isRootNetworksProcessing, setIsRootNetworksProcessing] = useState(false);
     const [isRootNetworkPanelMinimized, setIsRootNetworkPanelMinimized] = useState(false);
+    const isMonoRootStudy = useSelector((state: AppState) => state.isMonoRootStudy);
 
     const theme = useTheme();
-
     const panelStyle = useMemo(() => {
+        const widthSpacing = isRootNetworkPanelMinimized ? theme.spacing(22) : theme.spacing(38);
+        let minHeightSpacing;
+
+        if (isRootNetworkPanelMinimized) {
+            minHeightSpacing = isMonoRootStudy ? theme.spacing(6) : theme.spacing(12);
+        } else {
+            minHeightSpacing = isMonoRootStudy ? theme.spacing(14) : theme.spacing(38);
+        }
```

Comment:
It's getting hard to read and maintain
Can we make this parents fit it's children, and the children take some space ?

Each children would take necessary space, and the parent would not have to handle all those different dimensions